kajiki/xml_template.py

```diff
@@ -530,19 +530,16 @@ def __iter__(self):
 
     def _get_braced_expr(self):
         # see https://github.com/nandoflorestan/kajiki/pull/38
-        while True:
-            try:
-                compile(self.source[self.pos:], '', 'eval')
-            except IndentationError as e:
-                self.pos += 1
-                continue
-            except SyntaxError as se:
-                end = self.pos + sum([se.offset] + [len(line) + 1
-                                                    for idx, line in enumerate(self.source[self.pos:].splitlines())
-                                                    if idx < se.lineno - 1])
-                text = self.source[self.pos:end - 1]
-                self.pos = end
-                return self.expr(text)
+        try:
+            self.pos += len(self.source[self.pos:]) - len(self.source[self.pos:].lstrip())
```

Comment:
please assign `self.source[self.pos:]` to a variable that states what it represents (like `pycode_candidate` or something similar) so that we don't have to read `self.source[self.pos:]` over and over and in 6 months we will remember that it was expected to contain the python code.